flink-runtime/src/test/java/org/apache/flink/runtime/operators/hash/HashTablePerformanceComparison.java

```diff
@@ -102,7 +102,7 @@ public void testCompactingHashMapPerformance() {
 			System.out.println("Starting update...");
 			start = System.currentTimeMillis();
 			while(updater.next(target) != null) {
-				target.setValue(target.getValue()*-1);
```

Comment:
`target.getValue()` is always 0 at this point. (see the 6th paragraph in the PR opening comment)